packages/mobile/src/components/drawer/Drawer.tsx

```diff
@@ -35,52 +35,52 @@ const BACKGROUND_OPACITY = 0.5
 // Controls the amount of friction in swiping when overflowing up or down
 const OVERFLOW_FRICTION = 4
 
-export const useStyles = makeStyles(
-  ({ palette }, { zIndex = 5, shouldAnimateShadow = true }) => ({
-    drawer: {
-      backgroundColor: palette.neutralLight10,
-      position: 'absolute',
-      top: 0,
-      left: 0,
-      width: '100%',
-      elevation: zIndex,
-      zIndex,
-      shadowOpacity: shouldAnimateShadow ? 0 : MAX_SHADOW_OPACITY,
-      shadowRadius: 15,
-      borderTopRightRadius: BORDER_RADIUS,
-      borderTopLeftRadius: BORDER_RADIUS,
-      overflow: 'hidden'
-    },
-    fullDrawer: {
-      top: 0,
-      height: '100%'
-    },
-    content: {
-      height: 'auto'
-    },
-    fullScreenContent: {
-      height: '100%'
-    },
-    dismissContainer: {
-      position: 'absolute',
-      top: 24,
-      left: 24
-    },
-    background: {
-      position: 'absolute',
-      backgroundColor: 'black',
-      top: 0,
-      height: '100%',
-      width: '100%'
-    },
-
-    skirt: {
-      backgroundColor: palette.neutralLight10,
-      width: '100%',
-      height: 800
-    }
-  })
-)
+export const useStyles = makeStyles<
+  Pick<DrawerProps, 'zIndex' | 'shouldAnimateShadow'>
```

Comment:
personally when it gets to be pretty nested with types and props etc, making this it's own type, like StylesConfig or smtg, might be easier to read?